test/provision/compile.sh

```diff
@@ -29,5 +29,5 @@ else
         systemctl enable $service || echo "service $service failed"
         systemctl restart $service || echo "service $service failed to restart"
     done
+    sudo adduser vagrant cilium
```

Comment:
Please add a log message saying that you are performing this operation, e.g.:

```echo "running \"sudo adduser vagrant cilium\" "```

This will make it easier to track which commands have ran at a certain point in time.

Reply:
@ianvernon should be ok now.